src/lammpsio/snapshot.py

```diff
@@ -8,22 +8,35 @@
 
 
 class Snapshot:
-    """Particle configuration.
+    """
```

Comment:
Multiline docstrings should have a one-liner short description, followed by a blank line, followed by a longer description. Please revert here and elsewhere.

If you don't like the way the auto-generated documentation looks, check if you need a custom template like I wrote for `relentless`:
https://github.com/mphowardlab/relentless/blob/main/doc/source/_templates/autosummary/class.rst